src/test/java/uk/gov/crowncommercial/dts/scale/cat/controller/ProjectsControllerTest.java

```diff
@@ -190,12 +195,18 @@ void updateProcurementProjectName_200_OK() throws Exception {
    */
   @Test
   void listProcurementEventTypes_200_OK() throws Exception {
+
+    List<EventType> eventTypes = Arrays.stream(uk.gov.crowncommercial.dts.scale.cat.model.generated.DefineEventType.values())
+            .map(eventType -> new EventType(eventType.getValue(), eventType.getValue())).collect(Collectors.toList());
+    when(procurementProjectService.getEventTypes(PROC_PROJECT_ID))
+            .thenReturn(eventTypes);
+
     mockMvc
         .perform(get("/tenders/projects/" + PROC_PROJECT_ID + "/event-types")
             .with(validJwtReqPostProcessor).accept(APPLICATION_JSON))
         .andDo(print()).andExpect(status().isOk())
         .andExpect(content().contentType(APPLICATION_JSON)).andExpect(jsonPath("$.size()").value(5))
-        .andExpect(jsonPath("$[*]", contains("EOI", "RFI", "RFP", "DA", "SL")));
+        .andExpect(jsonPath("$[*]['type']", contains("EOI", "RFI", "RFP", "DA", "SL")));
```

Comment:
As explained above, the Tenders API endpoint `/tenders/projects/{proc-id}/event-types` returns an array of `DefineEventType`  - which does not have a type or description property